src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -762,23 +615,20 @@ private void analyzeUDist() {
 
         // Consecutive high UD - signifies disturbance event
         case CHUD:
-//            this.numMajorDist += 1;
 
           // Characterize the disturbance and following recovery
           setDisturbance(thisSeg.startYear, thisSeg.endYear);
```

Comment:
`this.numDist` is currently being incremented in setDisturbance at the very end of the function.  We had talked about bringing this outside each call to `setDisturbance`.  There are three such calls.  I've left it in `setDisturbance` for now.